src/components/identity/azure-identity.tsx

```diff
@@ -1,4 +1,4 @@
-import { Icon, Typography } from '@equinor/eds-core-react';
+import { Accordion, Icon, Typography } from '@equinor/eds-core-react';
```

Comment:
```suggestion
import { Icon, Typography } from '@equinor/eds-core-react';
```
Remove unused import